### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by csmeder**
Bernie I am having trouble getting Mocha running on my local, would you be willing to update this README with instructions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+**Roberto Iraheta**
+
++ http://github.com/riraheta
+
+Copyright and License
+---------------------
+
+Copyright 2012 Bernie Nowel 
+
+Licensed under ...
```
Comment (csmeder) [inline README.md:169]: Im thinking MIT or Apache 2.0 license, what do you guys think?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+ If you have OSX with home brew installed you can install node with the following command:
+
+    $ brew install node
+
+#### Configure Node.js
+
+    $ npm install
+
+Setting up Testing (Mocha) on your local
```
Comment (csmeder) [inline README.md:57]: Bernie, if you have time please update this section with the correct instructions for setting up and running Mocha

File: README.md (hunk 1 above)
Comment (bnowel) [inline README.md:169]: Not too worried about it.  Whatever license lets people do whatever they want with it.

File: README.md (hunk 2 above)
Comment (bnowel) [inline README.md:57]: You don't need to install Mocha globally, my guess is npm install doesn't automatically install the dev dependencies for you.  It's specified in package.json as a development dependency so it doesn't have to get installed when pushing to a production server.

**Pull request merged**